src/app/static/src/app/components/figures/strategise/util.ts

```diff
@@ -23,32 +23,25 @@ export const formatPercentage = (percentage: number) => new Intl.NumberFormat("e
 }).format(percentage);
 
 const interventionNames: Record<string, string> = {
-    "irs": "IRS* only",
+    "irs": "IRS only",
     "llin-pbo": "Pyrethroid-PBO ITN only",
-    "irs-llin-pbo": "Pyrethroid-PBO ITN with IRS*",
-    "llin": "Pyrethroid LLIN only",
-    "irs-llin": "Pyrethroid LLIN with IRS*",
+    "irs-llin-pbo": "Pyrethroid-PBO ITN with IRS",
+    "llin": "Pyrethroid-only ITN only",
+    "irs-llin": "Pyrethroid-only ITN with IRS",
+    "pyrrole-pbo": "Pyrethroid-pyrrole ITN only",
+    "irs-pyrrole-pbo": "Pyrethroid-pyrrole ITN with IRS",
     "none": "No intervention"
 };
 export const getInterventionName = (intervention: string) => interventionNames[intervention];
 
-// BTable uses Bootstrap colour variants for styling: https://bootstrap-vue.org/docs/components/table#items-record-data
-const interventionColourNames: Record<string, string> = {
-    "irs": "primary",
-    "llin-pbo": "secondary",
-    "irs-llin-pbo": "danger",
-    "llin": "success",
-    "irs-llin": "warning",
-    "none": ""
-};
-export const getInterventionColourName = (intervention: string) => interventionColourNames[intervention];
-
 const interventionColourValues: Record<string, string> = {
-    "irs": "dbb8db",
-    "llin-pbo": "e0fae0",
-    "irs-llin-pbo": "ffe6b8",
-    "llin": "bbf0fb",
-    "irs-llin": "f5c6cb",
+    "irs": "c080c0",
```

Comment:
Any way to easily sync these with `src/app/static/src/scss/partials/strategise.scss`? If not do you want to leave reciprocal comments indicating that updates should be reflected?

Reply:
Not trivially afaik but reciprocal comment is a good idea.